mkdocs.yml

```diff
@@ -37,13 +37,13 @@ extra:
       link: https://www.youtube.com/@inbibo9227/featured
     - icon: fontawesome/brands/facebook
       link: https://www.facebook.com/people/Inbibo/100064064923495/
-plugins:
-  - mkdocs-video:
-      is_video: True
-      video_muted: True
-      video_controls: True
-      css_style:
-        width: "75%"
+# plugins:
+#   - mkdocs-video:
+#       is_video: True
+#       video_muted: True
+#       video_controls: True
+#       css_style:
+#         width: "75%"
```

Comment:
Do we want to keep this commented for a specific reasson or should we remove it?

Reply:
I left it commented because it's a plugin that is not being used at the moment.  I can download the plugin so that it won't error tho and uncomment it.

Reply:
I uncommented.